webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubricStudentComment.js

```diff
@@ -49,7 +49,7 @@ export class SakaiRubricStudentComment extends RubricsElement {
 
         <div class="rubric-comment-dropdown dropdown-menu">
           <div class="m-2 rubric-comment-body">
-            <div class="fw-bold rubric-criterion-comment-title">${this.tr("comment_for_criterion", [ this.criterion.title ])}</div>
+            <div class="rubric-criterion-comment-title">${this.tr("comment_for_criterion", [ this.criterion.title ])}</div>
```

Comment:
I thought you were doing this task component by component? if you do them all like this QA will run a mile :) Too much to test at one time.

Reply:
Don't worry, I'll discuss it later in the Core Team call as well. You can count on the QA of all this by the Spanish institutions, we are very interested in having Sakai 25 to be able to migrate this summer.
I have to do a last review, but I think that all the changes that imply something for the corporate image are already in place.
It worries me a little that Codacy marks me as error those lines, since I am testing it and it works, it is a false positive, I think.